src/main/java/com/exadel/backendservice/entity/EventEntity.java

```diff
@@ -0,0 +1,57 @@
+package com.exadel.backendservice.entity;
+
+import com.exadel.backendservice.model.EventType;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+import java.sql.Timestamp;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Table(name = "event_table")
+@Data
+@NoArgsConstructor
+public class EventEntity {
```

Comment:
please rename to 'Event'. Or any other name without 'entity' postfix. It's redundant, I think

Reply:
corrected